cs4090/accounts/models/user.py

```diff
@@ -11,41 +11,56 @@
 class UserManager(BaseUserManager):
     """User manager"""
 
-    def _create_user(self, email, password=None, **extra_fields):
+    print("callled")
+
+    def _create_user(
+        self, email, password=None, first_name=None, last_name=None, **extra_fields
+    ):
         """Creates and returns a new user using an email address"""
         if not email:  # check for an empty email
             raise AttributeError("User must set an email address")
         else:  # normalizes the provided email
             email = self.normalize_email(email)
 
         # create user
-        user = self.model(email=email, **extra_fields)
+        user = self.model(
+            email=email, first_name=first_name, last_name=last_name, **extra_fields
+        )
         user.set_password(password)  # hashes/encrypts password
         user.save(using=self._db)  # safe for multiple databases
         return user
 
-    def create_user(self, email, password=None, **extra_fields):
+    def create_user(
+        self, email, password=None, first_name=None, last_name=None, **extra_fields
+    ):
         """Creates and returns a new user using an email address"""
         extra_fields.setdefault("is_staff", False)
         extra_fields.setdefault("is_superuser", False)
-        return self._create_user(email, password, **extra_fields)
+        return self._create_user(email, password, first_name, last_name, **extra_fields)
 
-    def create_staffuser(self, email, password=None, **extra_fields):
+    def create_staffuser(
+        self, email, password=None, first_name=None, last_name=None, **extra_fields
+    ):
         """Creates and returns a new staffuser using an email address"""
         extra_fields.setdefault("is_staff", True)
         extra_fields.setdefault("is_superuser", False)
-        return self._create_user(email, password, **extra_fields)
+        return self._create_user(email, password, first_name, last_name, **extra_fields)
 
-    def create_superuser(self, email, password=None, **extra_fields):
+    def create_superuser(
+        self, email, password=None, first_name=None, last_name=None, **extra_fields
+    ):
         """Creates and returns a new superuser using an email address"""
         extra_fields.setdefault("is_staff", True)
         extra_fields.setdefault("is_superuser", True)
-        return self._create_user(email, password, **extra_fields)
+        return self._create_user(email, password, first_name, last_name, **extra_fields)
 
 
 class User(AbstractBaseUser, PermissionsMixin):
     """Custom user model"""
 
+    print("callled")
```

Comment:
I would CTRL+SHIFT+F and remove any debug statements that we don't need anymore.